src/qt/recentrequeststablemodel.cpp

```diff
@@ -10,18 +10,20 @@
 #include <qt/walletmodel.h>
 
 #include <clientversion.h>
+#include <key_io.h>
```

Comment:
nit, sort 🙈

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18608#discussion_r453369490

> nit, sort

Thanks. Wonder if I'm slowly forgetting the alphabet.